plugin/src/index.ts

```diff
@@ -145,6 +145,18 @@ const plugin: NetlifyPlugin = {
       await writeEdgeFunctions(netlifyConfig)
       await updateConfig(publish)
     }
+
+    const middlewareManifest = await loadMiddlewareManifest(netlifyConfig)
+
+    if (!process.env.NEXT_USE_NETLIFY_EDGE && middlewareManifest.sortedMiddleware.length !== 0) {
+      console.log(
+        yellowBright(outdent`
+          You are using Next.js Middleware without Netlify Edge Functions.
+          Your middleware is running at origin, which disables static pages.
```

Comment:
```suggestion
          This will soon be deprecated because it negatively affects performance and will disable ISR and static rendering.
```
I don't think people will understand what running at the origin means